table/block_based/partitioned_index_reader.cc

```diff
@@ -148,6 +149,11 @@ void PartitionIndexReader::CacheDependencies(bool pin) {
   rep->CreateFilePrefetchBuffer(0, 0, &prefetch_buffer);
   s = prefetch_buffer->Prefetch(rep->file.get(), prefetch_off,
                                 static_cast<size_t>(prefetch_len));
+#ifndef NDEBUG
+  if (!s.ok()) {
+    TEST_SYNC_POINT("FaultInjectionIgnoredError");
```

Comment:
The name of the sync point seems incorrect here (ignored vs ignore); also I think we could use the macro in this file as well (in all three places).

Reply:
Good catch! Thanks. Somehow I missed this file.